Helm/opensearch/values.yaml

```diff
@@ -0,0 +1,315 @@
+---
+clusterName: "opensearch-cluster"
+nodeGroup: "master"
+
+# The service that non master groups will try to connect to when joining the cluster
+# This should be set to clusterName + "-" + nodeGroup for your master group
+masterService: "opensearch-cluster-master"
+
+# OpenSearch roles that will be applied to this nodeGroup
+# These will be set as environment variables. E.g. node.master=true
+roles:
+  master: "true"
+  ingest: "true"
+  data: "true"
+  remote_cluster_client: "true"
+
+replicas: 2
+minimumMasterNodes: 1
+
+majorVersion: ""
+
+
+# Allows you to add any config files in /usr/share/opensearch/config/
+# such as opensearch.yml and log4j2.properties
+config:
+#  opensearch.yml: |
+#    xpack.security.enabled: false
+#    xpack.security.transport.ssl.enabled: false
+#    xpack.security.transport.ssl.verification_mode: certificate
+#    xpack.security.transport.ssl.keystore.path: /usr/share/opensearch/config/certs/opensearch-certificates.p12
+#    xpack.security.transport.ssl.truststore.path: /usr/share/opensearch/config/certs/opensearch-certificates.p12
+#    xpack.security.http.ssl.enabled: false
+#    xpack.security.http.ssl.truststore.path: /usr/share/opensearch/config/certs/opensearch-certificates.p12
+#    xpack.security.http.ssl.keystore.path: /usr/share/opensearch/config/certs/opensearch-certificates.p12
```

Comment:
@TheAlgo I think we should make this YAML based, not templated from a string. You lose a lot of flexibility in making this a string. 

Especially considering the config is a YAML. I'm going to make a PR for this.

Reply:
PR made which includes this.

Reply:
Thanks Aaron, I have merged the PR.